canonical-web-and-design/manager.assets.ubuntu.com
# Add sso to Assets Manager

# QA

Run the server, observe that all endpoints (`index, create, update`) now require launchpad auth, specifically membership of the `canonical-content-people` team.
